docs/api/README.md

```diff
@@ -0,0 +1,190 @@
+# Flipper::Api
+
+API for the [Flipper](https://github.com/jnunemaker/flipper) gem.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+    gem 'flipper-api'
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install flipper-api
+
+## Endpoints
+
+### Get all features
+
+`GET /flipper-api/api/v1/features`
```

Comment:
👍  I really like this structure. I left the `URL` section and added cURL examples for each endpoint.